docs/user/kafka/confluent.rst

```diff
@@ -7,11 +7,189 @@ Confluent Integration
 
   Confluent integration is currently experimental and supports consuming from Kafka, but not producing.
 
-The Kafka data store can integrate with Confluent Kafka topics and the Confluent Schema Registry. The schema
-registry is a centralized store of versioned Avro schemas, each associated with a particular Kafka topic.
+The Kafka Data Store can integrate with Confluent Kafka topics and the Confluent Schema Registry. The schema
+registry is a centralized store of versioned Avro schemas, each associated with a particular Kafka topic. The
+``ConfluentKafkaDataStore`` can convert an Avro schema into a ``SimpleFeatureType`` and seamlessly deserialize
+records into ``SimpleFeatures``.
 
 To read from a Confluent topic, set the URL to the schema registry in your data store parameter map under the key
 ``kafka.schema.registry.url``. In GeoServer, select the "Confluent Kafka (GeoMesa)" store instead of the
 regular Kafka store.
 
 Note that Confluent requires Avro 1.8 and the Confluent client JARs, which are not bundled with GeoMesa.
+
+Supported Avro Schema Fields
+----------------------------
+
+The following avro schema field types are supported: ``STRING``, ``BOOLEAN``, ``INT``, ``DOUBLE``, ``LONG``, ``FLOAT``,
+``BYTES``, and ``ENUM``. The ``UNION`` type is supported only for unions of null and one other supported type,
+e.g. ``"type": ["null","string"]``.
+
+Supported ``SimpleFeature`` Fields
+----------------------------------
+
+To encode types that may be part of a ``SimpleFeature``, but are not part of a standard Avro schema, e.g. ``Geometry``
+or ``Date``, the Confluent Kafka Data Store supports interpreting several key-value metadata attributes on schema
+fields, which are described below.
+
+All the attribute values are case insensitive.
+
+``geomesa.geom.format``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Indicates that the field should be interpreted as a ``Geometry`` in the given format. Must be accompanied by the key
+``geomesa.geom.type``.
+
+=========== ===================== ====================================================
+Value       Schema Field Type     Description
+=========== ===================== ====================================================
+``wkt``     ``STRING``            Well-Known Text representation of a ``Geometry``
+``wkb``     ``BYTES``             Well-Known Binary representation of a ``Geometry``
```

Comment:
Do we want to support geojson geometries or TWKB in this PR?

Reply:
hmm, i would say not. does it seem likely that people would be putting geojson onto kafka topics over WKT? TWKB would be a nice to have, but probably not widely used.

Reply:
The GeoJSON use case may be that folks would send the whole feature as GeoJSON, but that's a separate thing.